### DISCUSSION TITLE
Get testfile from context

### DISCUSSION THREAD

**Discussion by servohatred** [artilleryio/artillery#2629] [Artillery]
Is there a way to get the yaml test file path that is currently being used by the test? Maybe from the context?

**Comment by hassy** [comment]
there isn't at the moment @servohatred but we can look at adding that /cc @bernardobridge

**Comment by bernardobridge** [comment]
Actually @servohatred , I'm looking into something very similar to this, I'll see what can be done there. What's your use-case , out of curiosity?
